dbcon/joblist/tuplehashjoin.h

```diff
@@ -614,10 +611,19 @@ class TupleHashJoinStep : public JobStep, public TupleDeliveryStep
     std::vector<boost::shared_ptr<joiner::TupleJoiner> > tbpsJoiners;
     std::vector<boost::shared_ptr<joiner::TupleJoiner> > djsJoiners;
     std::vector<int> djsJoinerMap;
-    boost::scoped_array<uint64_t> memUsedByEachJoin;
+    boost::scoped_array<ssize_t> memUsedByEachJoin;
```

Comment:
Why not use here unique_ptr for the vector? It [seems](http://boost.2283326.n4.nabble.com/shared-array-Why-not-in-C-11-td4649418.html) that scoped_array never made into standart b/c there was no reason for this specific structure.

Reply:
It could be a plain vector.  I left it how it was.  Not that important.